packages/bluebird-icons/scripts/generate-index.js

```diff
@@ -0,0 +1,34 @@
+const fs = require('fs');
+const path = require('path');
+
+const dir = path.join(__dirname, '../src/components');
+const outputFilePath = path.join(__dirname, '../src/index.js');
+
+/**
+ * TODO: future we could run svgr to generate the components into `./src/components`.
+ * Today this is handled by `yarn -> @svgr/cli`
+ */
+
+/**
+ * Generate a new lib index.js based on components in `lib/components`
+ */
+fs.readdir(dir, (err, componentsFileNames) => {
+  if (!err) {
+    // gather file contents from components directory
+    const fileContents = componentsFileNames.reduce((a, componentFileName) => {
+      return (a += `export ${
+        componentFileName.split('.')[0]
+      } from './components/${componentFileName}';\n`);
+    }, '');
+
+    console.log('fileContents', fileContents);
```

Comment:
Is there a reason we're leaving this log in? Doesn't really feel like we need it.

Reply:
It's primarily a progress indicator, to provide the CLI with output as progress is being made (so the user is aware that something is happening between script start and sucess on line `29`.